sdk-extensions/metric-incubator/README.md

```diff
@@ -0,0 +1,77 @@
+# OpenTelemetry Metric Incubator
+
+[![Javadocs][javadoc-image]][javadoc-url]
+
+This artifact contains experimental code related to metrics.
+
+## View File Configuration
+
+Adds support for file based YAML configuration of Metric SDK Views.
+
+For example, suppose `/Users/user123/view.yaml` has the following content:
+
+```yaml
+- selector:
+    instrument_name: my-instrument
+    instrument_type: COUNTER
+    meter_name: my-meter
+    meter_version: 1.0.0
+    meter_schema_url: http://example.com
+  view:
+    name: new-instrument-name
+    description: new-description
+    aggregation: histogram
+    attribute_keys:
+      - foo
+      - bar
+```
+
+The equivalent view configuration would be:
+
+```
+SdkMeterProvider.builder()
+   .registerView(
+       InstrumentSelector.builder()
+           .setInstrumentName("my-instrument")
+           .setInstrumentType(InstrumentType.COUNTER)
+           .setMeterSelector(
+               MeterSelector.builder()
+                   .setName("my-meter")
+                   .setVersion("1.0.0")
+                   .setSchemaUrl("http://example.com")
+                   .build())
+           .build(),
+       View.builder()
+           .setName("new-instrument")
+           .setDescription("new-description")
+           .setAggregation(Aggregation.histogram())
+           .filterAttributes(key -> new HashSet<>(Arrays.asList("foo", "bar")).contains(key))
+           .build());
+```
+
+If using [autoconfigure](../autoconfigure) with this artifact on your classpath, it will automatically load a list of view config files specified via environment variable or system property:
+
+| System property                      | Environment variable                 | Purpose                                                    |
+|--------------------------------------|--------------------------------------|------------------------------------------------------------|
+| otel.experimental.metric.view.config | OTEL_EXPERIMENTAL_METRIC_VIEW_CONFIG | List of absolute files containing view configuration YAML. |
```

Comment:
```suggestion
| otel.experimental.metrics.view_config | OTEL_EXPERIMENTAL_METRIC_VIEW_CONFIG | List of absolute paths to files containing view configuration YAML. |
```

Reply:
How about `otel.experimental.metrics.view-config` for consistency with [otel.java.disabled.resource-providers](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#disabling-automatic-resourceproviders)?